### DISCUSSION TITLE
RTL Langauge Support

### DISCUSSION THREAD

**Discussion by Fedora-Man** [holger1411/understrap#1672] [Q&A]
How to add support for RTL languages such as Arabic and apply different styles when in RTL?

**Comment by bacoords** [comment]
Here's a great starting point for using RTL with Bootstrap: 
https://getbootstrap.com/docs/5.1/getting-started/rtl/

**Reply by Fedora-Man** [reply]
That is great! but how to integrate bootstrap-rtl with understrap?

**Reply by bacoords** [reply]
Here's an additional resource on how to enqueue RTL assets in your theme 
https://codex.wordpress.org/Right_to_Left_Language_Support

**Reply by Fedora-Man** [reply]
It turns out that bootstrap-rtl is an alternative version of bootstrap and is processed/complied with RTLCSS:
https://stackoverflow.com/questions/67055428/how-to-enable-bootstrap-rtl-in-scss

Is there a way to update understrap for this?

**Reply by bacoords** [reply]
Yes, I would recommend making a child theme specifically for RTL. You'll need to look at their build process- you can look at how Bootstrap runs a separate [`css-rtl`](https://github.com/twbs/bootstrap/blob/main/package.json#L49) commands for compiling RTL stylesheets. You can probably try to copy some of their build process into your Understrap child theme.
